x-pack/plugins/security_solution/public/flyout/document_details/right/index.tsx

```diff
@@ -58,7 +58,17 @@ export const RightPanel: FC<Partial<RightPanelProps>> = memo(({ path }) => {
     const defaultTab = tabsDisplayed[0].id;
     const tabSavedInlocalStorage = storage.get(FLYOUT_STORAGE_KEYS.RIGHT_PANEL_SELECTED_TABS);
 
-    if (!path) return tabSavedInlocalStorage || defaultTab;
+    if (!path) {
```

Comment:
I'm considering wrapping this whole `useMemo` (and the previous one calculating the `tabDisplayed`) into a hook to make this unit testable.
What do you think @christineweng

Reply:
@christineweng I implemented the hook and its unit test in this [last commit](https://github.com/elastic/kibana/pull/180391/commits/e9d7f1b023794085be28368a54b329c8d5cf3f43)